photon-api/src/main/scala/com/linkedin/photon/ml/optimization/game/RandomEffectOptimizationProblem.scala

```diff
@@ -43,27 +43,57 @@ protected[ml] class RandomEffectOptimizationProblem[Objective <: SingleNodeObjec
     val isTrackingState: Boolean)
   extends RDDLike {
 
+  /**
+   * Get the Spark context.
+   *
+   * @return The Spark context
+   */
   def sparkContext: SparkContext = optimizationProblems.sparkContext
```

Comment:
I have a strong rejection to having a method called `sc`. If I had it my way, I would have the Spark guys rename the members to `sparkContext` as well. One thing to note is that in the actual `RDD` source, they do what I mentioned above: member = `sc`, method = `sparkContext`.

You have a good point about consistency, but the Spark guys aren't consistent themselves and I hate the idea of a method named `sc`...